src/flowMC/sampler/Sampler.py

```diff
@@ -113,11 +128,16 @@ def __init__(
         production["log_prob"] = jnp.empty((self.n_chains, 0))
         production["local_accs"] = jnp.empty((self.n_chains, 0))
         production["global_accs"] = jnp.empty((self.n_chains, 0))
+        
+        if self.track_gelman_rubin:
```

Comment:
Since R_hat can be compute in post-processing, I don't think we should have it in the main sampler.